packages/analytics/index.test.ts

```diff
@@ -179,20 +272,22 @@ describe('FirebaseAnalytics instance tests', () => {
   });
 
   describe('Page has no existing gtag script or dataLayer', () => {
-    before(() => {
+    before(async () => {
       resetGlobalVars();
-      const app = getFakeApp(analyticsId);
+      const app = getFakeApp(fakeAppParams);
       const installations = getFakeInstallations();
+      stubFetch(200, {});
       analyticsInstance = analyticsFactory(app, installations);
     });
     after(() => {
       delete window['gtag'];
       delete window['dataLayer'];
       removeGtagScript();
+      fetchStub.restore();
     });
     it('Adds the script tag to the page', async () => {
-      const { initializedIdPromisesMap } = getGlobalVars();
-      await initializedIdPromisesMap[analyticsId];
+      const { initializationPromisesMap } = getGlobalVars();
+      await initializationPromisesMap[fakeMeasurementId];
```

Comment:
Isn't the key for `initializationPromisesMap` appId? Using `fakeMeasurementId` seems wrong if it's the case.

Reply:
Ack, yes, fixed.